src/users/users.service.ts

```diff
@@ -0,0 +1,58 @@
+import {Injectable, NotFoundException} from '@nestjs/common';
+import {CreateUserDto} from './dto/create-user.dto';
+import {UpdateUserDto} from './dto/update-user.dto';
+import {IUser} from "./interfaces/user.interface";
+import {PartialUpdateUserDto} from "./dto/partial-update-user.dto";
+
+@Injectable()
+export class UsersService {
+    private mockUsers: (IUser)[] = []
```

Comment:
it could be:
private mockUsers: IUser[] = []
No need to have it in brackets

Reply:
Ok, i understand!
Thanks